python/src/aac/plugins/lsp_server/providers/rename_provider.py

```diff
@@ -80,6 +80,13 @@ def _get_rename_edits(self, symbol: str, new_name: str) -> dict[str, TextEdit]:
             else:
                 edits = self._get_definition_name_text_edits(new_name, definition_to_find, language_context)
 
+        if SymbolType.ENUM_VALUE_TYPE in symbol_types:
+            enum_to_find = language_context.get_enum_definition_by_type(name)
+            if not enum_to_find:
+                logging.warn(f"Can't find references for non-enum-type {name}")
```

Comment:
I think we can add a little more context.
```suggestion
                logging.warn(f"{name} was not found in the context, couldn't find references to it.")
```